packages/amplify-cli/src/extensions/amplify-helpers/editor-selection.ts

```diff
@@ -2,11 +2,19 @@ import * as inquirer from 'inquirer';
 import { JSONUtilities } from 'amplify-cli-core';
 import { merge } from 'lodash';
 
-export const editors = [
+const editors = [
   {
     name: 'Visual Studio Code',
     value: 'vscode',
   },
+  {
+    name: 'Android Studio',
+    value: 'android-studio',
+  },
+  {
+    name: 'Xcode (Mac OS only)',
```

Comment:
nit could have some logic to concat this option to the list only if process.platform() === 'darwin'

Reply:
Just following the pattern we use, not refactoring the behavior in this PR.